source/utilities.tex

```diff
@@ -10821,6 +10821,23 @@
     struct greater_equal;                                                           // freestanding
     struct less_equal;                                                              // freestanding
   }
+
+  template<class Fn, class... Args>
+    concept @\defexposconcept{callable}@ =         // \expos
+      requires (Fn&& fn, Args&&... args) {
+        std::forward<Fn>(fn)(std::forward<Args>(args)...);
+      };
+  template<class Fn, class... Args>
+    concept @\defexposconcept{nothrow-callable}@ =   // \expos
+      \exposconcept{callable}<Fn, Args...> &&
+      requires (Fn&& fn, Args&&... args) {
+        { std::forward<Fn>(fn)(std::forward<Args>(args)...) } noexcept;
+      };
+  template<class Fn, class... Args>
+    using @\exposid{call-result-t}@ = decltype(declval<Fn>()(declval<Args>()...));  // \expos
+
+  template<const auto& T>
+    using @\exposid{decayed-typeof}@ = decltype(auto(T));                           // \expos
```

Comment:
Can we just reuse the existing `decay_t`? (Perhaps changes should be in another PR, although.)